card/lib/card/view/cache/stub.rb

```diff
@@ -9,30 +11,11 @@ module Cache
       #
       # A stub must contain all the information necessary to produce the view as intended.
       module Stub
-        class << self
-          def escape stub_json
-            stub_json.gsub "(", "_OParEN_"
-          end
-
-          def unescape stub_json
-            stub_json.gsub "_OParEN_", "("
-          end
-
-          # FIXME: escaping and unescaping stubs should not be necessary
-          # It's used to avoid problems with altered views, but altering views is
-          # unsafe and should be eliminated.  See {Card::View::Cache}.
```

Comment:
I don't think this comment is correct. I think the reason we had to escape/unescape parens was that we were matching on `[^)]` in view_stub.rb